t/23-amqp.t

```diff
@@ -334,4 +334,16 @@ subtest 'extra TLS query params' => sub {
     is($last_publisher->url, $expected, 'url has all params');
 };
 
+subtest 'enqueue_restart triggers amqp event and log' => sub {
+    my $test_job = $schema->resultset('Jobs')->find(99945);
+    my $minion_id;
+    %published = ();    # just to be sure
+    combined_like { $minion_id = $test_job->enqueue_restart }
+    qr/Enqueued restarting openQA job 99945 via Minion job \d+/, 'Debug log for enqueue_restart is present';
+    ok $minion_id, 'Got a Minion job ID back';
+    # each event contains keys(headers args body)
```

Comment:
I would remove this comment. If this is important it should be checked with code.